### ISSUE TITLE
Undefined state

### ISSUE THREAD

**Issue by stappersg** [radvd-project/radvd#138]
This issue is for expressing my concern of the undefined state this project is in.

Executive summary:    No one is clearly saying  _I take the lead for the time being_

There is #135 with some vague update _yes,  a rename did happen_.  Nowhere is any sign of whom did that.

Merge request #136  is not yet touched by some one who has merge privilege.

The thing that makes me most uncomfortable is that https://github.com/orgs/radvd-project/people is empty.

**Comment by Neustradamus** [comment]
@stappersg:
1/ @reubenhwk stops the development of radvd: https://github.com/radvd-project/radvd/issues/135
2/ Reuben has moved radvd and radvdweb into the radvd-project organization
2/ I have sent you 2 emails the 9 January 2021 (the day of the announcement of Reuben retirement) and one new 11 January 2021 in more https://github.com/radvd-project/radvd/issues/135#issuecomment-757348031 + https://github.com/radvd-project/radvd/issues/125 + https://github.com/radvd-project/radvd/issues/129
3/ You have received an invitation to join the radvd-project organization for your work into Debian. No validation at this time.
4/ Like you can see: https://github.com/radvd-project: "Official radvd organization" /  https://radvd.litech.org/ "Verified"
5/ By default, GitHub does not show people in organization
6/ @robbat2 has signed 2.19 builds: https://github.com/radvd-project/radvd/issues/125 + https://github.com/radvd-project/radvd/issues/129

**Comment by robbat2** [comment]
@stappersg I marked myself as public in the organization for you.

While I'm willing to take ownership of the project, I'm not a fan of the BDFL model overall, and it certainly shouldn't exempt anybody, including myself from having our changes extensively reviewed. 

As I wrote on #136, I'm looking for more clear reviews before I click the merge button.

As I wrote in #129, if @reubenhwk hadn't lost their old GPG keys, then they could have signed a message stating that myself and @Neustradamus were his picks to take over the project.
